pkg/cmd/ctlstore/main.go

```diff
@@ -79,13 +79,14 @@ type executiveCliConfig struct {
 // running its own reflector.  The LDBPath will come from the composed
 // reflector config instead of being a top level element in this struct.
 type supervisorCliConfig struct {
-	SnapshotInterval    time.Duration      `conf:"snapshot-interval" help:"Wait time between snapshots" validate:"nonzero"`
-	SnapshotURL         string             `conf:"snapshot-url" help:"URL for snapshot upload (i.e. s3://bucket/key)" validate:"nonzero"`
-	Debug               bool               `conf:"debug" help:"Turns on debug logging"`
-	LedgerLatencyConfig ledgerHealthConfig `conf:"ledger-latency-health" help:"Configures ledger latency health behavior"`
-	ReflectorConfig     reflectorCliConfig `conf:"reflector" help:"reflector configuration"`
-	Shadow              bool               `conf:"shadow" help:"set this to true to emit shadow=true metric tags"`
-	Dogstatsd           dogstatsdConfig    `conf:"dogstatsd" help:"dogstatsd Configuration"`
+	SnapshotInterval     time.Duration      `conf:"snapshot-interval" help:"Wait time between snapshots" validate:"nonzero"`
+	SnapshotURL          string             `conf:"snapshot-url" help:"URL for snapshot upload (i.e. s3://bucket/key)" validate:"nonzero"`
+	Debug                bool               `conf:"debug" help:"Turns on debug logging"`
+	LedgerLatencyConfig  ledgerHealthConfig `conf:"ledger-latency-health" help:"Configures ledger latency health behavior"`
+	ReflectorConfig      reflectorCliConfig `conf:"reflector" help:"reflector configuration"`
+	Shadow               bool               `conf:"shadow" help:"set this to true to emit shadow=true metric tags"`
+	Dogstatsd            dogstatsdConfig    `conf:"dogstatsd" help:"dogstatsd Configuration"`
+	MinimumLedgerLatency time.Duration      `conf:"minimum-ledger-latency" help:"Minimum ledger latency, at which the supervisor halts snapshots and waits for the ledger to catch up."`
```

Comment:
Would `MaxPublishableLedgerLatency` be a better variable name here?

Reply:
🤦‍♂ should be `Maximum`! I tend to prefer more conciseness here, ala Go style :)